proto/wechaty/puppet/base.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
-package wechaty.puppet.base;
+package wechaty.puppet;
+
+option go_package="github.com/wechaty/go-wechaty/wechaty-puppet-hostie/generate/wechaty/puppet";
```

Comment:
I'd like to publish the Go version of GRPC at <github.com/chatie/grpc/go/wechaty-puppet>, how do you think about that?

Reply:
You can put * .pb.go generated by .proto in <github.com/chatie/grpc/go/wechaty-puppet>, go-wechaty can be used directly by import.